test/test_ops.py

```diff
@@ -362,27 +385,44 @@ def test_python_reference_consistency(self, device, dtype, op):
 
             if isinstance(actual, torch.Tensor):
                 assert isinstance(expected, torch.Tensor)
-                prims.utils.compare_significant_strides(actual, expected)
+                prims.utils.compare_tensor_meta(actual, expected)
                 if getattr(op, 'validate_view_consistency', True):
                     self.assertEqual(actual._is_view(), expected._is_view())
             if isinstance(actual, Sequence):
                 assert isinstance(expected, Sequence)
                 for a, b in zip(actual, expected):
-                    prims.utils.compare_significant_strides(a, b)
+                    prims.utils.compare_tensor_meta(a, b)
                     if getattr(op, 'validate_view_consistency', True):
                         self.assertEqual(a._is_view(), b._is_view())
 
 
     @skipMeta
     @onlyNativeDeviceTypes
-    @ops([op for op in ops_and_refs if op.error_inputs_func is not None], dtypes=OpDTypes.none)
+    @ops([op for op in op_db if op.error_inputs_func is not None], dtypes=OpDTypes.none)
     def test_errors(self, device, op):
         error_inputs = op.error_inputs(device)
         for ei in error_inputs:
             si = ei.sample_input
             with self.assertRaisesRegex(ei.error_type, ei.error_regex):
                 op(si.input, *si.args, **si.kwargs)
 
+    @skipMeta
+    @onlyNativeDeviceTypes
+    @ops([op for op in python_ref_db if op.error_inputs_func is not None], dtypes=OpDTypes.none)
+    def test_python_reference_errors(self, device, op):
+        def _to_tensormeta(x):
+            if isinstance(x, torch.Tensor):
+                return prims.utils.TensorMeta(x)
+            return x
+
+        error_inputs = op.error_inputs(device)
+        for ei in error_inputs:
+            si = ei.sample_input
+            meta_sample = si.transform(_to_tensormeta)
+            # TODO: match strings
```

Comment:
expect tests would be very helpful here, then you wouldn't have to manually type in the correct strings everywhere